app/actions.tsx

```
@@ -277,202 +157,81 @@ export const AI = createAI<AIState, UIState>({
+    const { role, content, id, type, name } = message;
+
+    switch (role) {
+      case 'user':
+        return {
+          id,
+          component: <UserMessage message={content} />,
+        };
+      case 'assistant':
+        switch (type) {
+          case 'response':
+            const streamableContent = createStreamableValue();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)